### PR TITLE
Skil 183

### PR DESCRIPTION
**Pull request by brianlugo1**
The Great Merge!!!

### CODE REVIEW

**Comment by brianlugo1** [comment]
@sah0017 Currently to my knowledge the only functionality that I have not fully tested is Student and Team Bulk Upload! Every other functionality I have tested and seems to be working as should. Although of course there is still the chance that I have missed some bugs!

**Review by Wrimo — APPROVED** [review]
LGTM!

File: BackEndFlask/Functions/teamImport.py [hunk 1]
```diff
@@ -30,168 +30,112 @@
...
             reader = itertools.tee(csv.reader(teamcsv))[0]
             for row in reader:
                 rowList = list(row)
                 team_name = rowList[0].strip()
                 ta_email = rowList[1].strip()
                 missingTA = False
-                if ' ' in ta_email or '@' not in ta_email or not isValidEmail(ta_email):
+                if not isValidEmail(ta_email):
                     delete_xlsx(teamFile, isXlsx)
                     return SuspectedMisformatting.error
```
[the remark this answers is not in the document]
Comment (Wrimo) [reply BackEndFlask/Functions/teamImport.py]: I fixed this.

**Comment by brianlugo1** [comment]
@sah0017 I am going to go ahead and merge this pull request with only one approved code review!

**Pull request merged**